views/splash.html

```diff
@@ -0,0 +1,95 @@
+    <title>{% if title %}{{ title }} :: {% endif %}Developers' Guild</title>
+    <link rel="stylesheet" href="../css/font-awesome.min.css" type="text/css">
+    <link rel="stylesheet" href="../css/semantic.min.css" type="text/css">
+    <link href='https://api.tiles.mapbox.com/mapbox.js/v2.1.8/mapbox.css' rel='stylesheet' />
+    <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="../css/style.css" type="text/css" media="screen, projection">
+</head>
+
+<body>
+    <div class="splash">
+      <div class="logo">
+        <img src="/images/daze-white_1024.png">
+        <h1>DAZE</h1>
+      </div>
+      <div class="tagline">Campus Community App</br>for De Anza College</div>
+      <div class="btn-menu">
+        <a href="home" class="btn">ENTER</a>
+        <a href="#" id="login-btn" class="btn secondary">LOGIN</a>
+        <a href="#" id="join-btn" class="btn secondary">JOIN</a>
+      </div>
+    </div>
+
+    <div class="ui basic modal login">
+      <i class="close fa fa-close"></i>
+      <div class="header">
+        Login 
+      </div>
+      <div class="content">
+        <div class="ui form">
+          <div class="field">
+              <label>Username</label>
+              <input type="text">
+            </div>
+            <div class="field">
+              <label>Password</label>
+              <input type="password">
+            </div>
+        </div>
+      </div>
+      <div class="actions">
+        <div class="two fluid ui inverted buttons">
+          <div class="ui green basic inverted button">
+            <i class="fa fa-lock"></i>
+            Login
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="ui basic modal join">
+      <i class="close fa fa-close"></i>
+      <div class="header">
+        Join
+      </div>
+      <div class="content">
+        <div class="ui form">
+          <div class="field">
+            <label>Username</label>
+            <input type="text">
+          </div>
+          <div class="field">
+            <label>Password</label>
+            <input type="password">
+          </div>
+          <div class="field">
+            <label>Email</label>
+            <input type="email">
+          </div>
+        </div>
+      </div>
+      <div class="actions">
+        <div class="two fluid ui inverted buttons">
+          <div class="ui green basic inverted button">
+            <i class="fa fa-lock"></i>
+            Register
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <script src="../js/jquery-1.10.2.min.js"></script>
+    <script src="../js/semantic.min.js"></script>
+    <script src="../js/scripts.js"></script>
+    <script src='https://api.tiles.mapbox.com/mapbox.js/v2.1.8/mapbox.js'></script>
+    <script type="text/javascript">
+      $('.modal').modal('attach events', '#login-btn', 'show');
```

Comment:
if i remember correctly, we only have one modal right now. but $('.modal') selects every div element with class 'modal', so it may have unexpected behaviors.